### PR TITLE
nuxt.js docs

### PR DESCRIPTION
**Pull request by daliborgogic**

### CODE REVIEW

File: content/nuxt.md [hunk 1]
```diff
@@ -3,4 +3,265 @@ path: "/docs/nuxt"
...
+const { GA } = process.env
+
+export default {
+  build: {
+    extend(config, ctx) {
+      if (ctx.isClient) {
+        const guessOptions = {
+          runtime: {
+            delegate: true,
+            prefetchConfig: {
```
Comment (mgechev) [inline content/nuxt.md]: We can drop `prefetchConfig`, `delegate`, and `routeProvider`. The default values of the last two are `true` and `false` respectively, and we don't have to update the thresholds.

File: content/nuxt.md [hunk 2]
```diff
@@ -3,4 +3,265 @@ path: "/docs/nuxt"
...
+        config.plugins.push(
+          new GuessPlugin(guessOptions)
+        )
+      }
+    }
+  }
+}
+```
+
+Alternative for alternatively, a  [@nuxtjs/guess](https://github.com/daliborgogic/guess-module) module for Guess.js!
```
Comment (mgechev) [inline content/nuxt.md]: Maybe you can add a section on how to use Guess.js with `@nuxtjs/guess`?

File: content/nuxt.md [hunk 3]
```diff
@@ -3,4 +3,265 @@ path: "/docs/nuxt"
...
+
+### Setup
+
+- Add `@nuxtjs/guess` dependency to your project
+- Add `@nuxtjs/guess` to modules section of `nuxt.config.js`
+
+```json
+{
+  modules: [
```
Comment (mgechev) [inline content/nuxt.md]: GitHub complains because the property names & values are not within double quotes.

**Review by mgechev — APPROVED** [review]
Great content! Thank you for the guide!

**Comment by daliborgogic** [comment]
You're welcome @mgechev.

**Pull request merged**